packages/scripts/src/admin-proposals/simulateVoteV2.ts

```diff
@@ -89,66 +101,76 @@ async function simulateVoteV2() {
   console.groupEnd();
   assert(votingStats.pendingRequests.length >= 1, "Must be at least 1 pending admin price request to vote on");
 
-  // Vote affirmatively from Foundation wallet.
-  const requestsToVoteOn: {
-    identifier: BytesLike;
-    time: BigNumberish;
-    salt: BigNumberish;
-    ancillaryData: BytesLike;
-    price: BigNumberish;
-    voteHash: BytesLike;
-  }[] = [];
+  const voterRequests: { [key: string]: VoteRequest[] } = {};
+  if (realVoters) {
+    // Fetch all voters.
+    const uniqueVoters = await getUniqueVoters(votingV2);
+    for (const voter of uniqueVoters) {
+      voterRequests[voter] = [];
+    }
+  } else {
+    // If not using real voters, then vote from the foundation wallet.
+    voterRequests[FOUNDATION_WALLET] = [];
+  }
+  // Set voter balances to 10 ETH for covering gas costs
+  for (const voter in voterRequests) {
+    await hre.network.provider.send("hardhat_setBalance", [voter, hre.ethers.utils.parseEther("10.0").toHexString()]);
```

Comment:
Nothing to change, but just curious if it's possible to set the base fee to 0 for a hardhat fork so that all transactions are free, so we could avoid having to make sure the addresses have ETH.

Reply:
Not sure, but good idea to investigate when we have to return to this issue.